style.css

```diff
@@ -0,0 +1,192 @@
+* {
+  font-family: "Inter", sans-serif;
+}
+
+.hero {
+  height: 100vh;
+  width: 100%;
+  min-height: 500px;
+  background-image: url("../images/background.png");
+  background-size: cover;
+  background-position: center;
+  background-color: rgb(255, 250, 250);
+  display: block;
+  align-items: center;
+  justify-items: center;
```

Comment:
- [OPTIONAL] Kindly make sure this container is set to `displayed:flex`  as recommended by the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/1_setup.md#:~:text=In%20order%20to%20lay%20out%20the%20elements%20on%20the%20page%20you%20should%20use%20Flexbox%20in%20all%202%20sections.) :factory_worker: 

__Please note that this option is crucial for the project, It's optional for now only because the previous reviewer didn't catch it__